config/install/migrate_plus.migration.joomla_node.yml

```diff
@@ -19,6 +19,17 @@ process:
   sticky:
     plugin: default_value
     default_value: 0
+  langcode:
+    plugin: default_value
+    source: language
+    default_value: "und"
+  status: state
+  created: created
```

Comment:
Drupal's created field will expect a UNIX timestamp. The create row in the source column provides a date.

Please see Dennis' implementation in the joomla_user migration for an example of the callback plugin you will need.